libshaderc/src/shaderc_test.cc

```diff
@@ -175,7 +180,8 @@ class CompileStringTest : public testing::Test {
                            options);
     EXPECT_TRUE(shaderc_module_get_success(comp.result())) << kind << '\n'
                                                            << shader;
-    return shaderc_module_get_bytes(comp.result());
+    return std::string(shaderc_module_get_bytes(comp.result()),
```

Comment:
If we don't use the 'length' to build the string and return char*, EXPECT_THAT(...HasSubstr()) will cut the output binary array when it sees a '\0' byte.